folly/CMakeLists.txt

```diff
@@ -57,6 +57,7 @@ list(REMOVE_ITEM files
   ${FOLLY_DIR}/experimental/exception_tracer/StackTrace.cpp
   ${FOLLY_DIR}/experimental/io/AsyncIO.cpp
   ${FOLLY_DIR}/experimental/io/HugePageUtil.cpp
+  ${FOLLY_DIR}/experimental/logging/example/
```

Comment:
`/experimental/logging/example/` should be added to the list passed to `HHVM_REMOVE_MATCHES_FROM_LISTS`, rather than here.